allennlp/__init__.py

```diff
@@ -2,27 +2,26 @@
 # (to avoid running into this bug: https://bugs.python.org/issue29246)
 import sys
 
-if sys.version_info < (3, 6, 1):
-    raise RuntimeError("AllenNLP requires Python 3.6.1 or later")
-
 # We get a lot of these spurious warnings,
 # see https://github.com/ContinuumIO/anaconda-issues/issues/6678
-import warnings  # noqa
+import warnings
+
+from allennlp.version import VERSION as __version__  # noqa
```

Comment:
Does this one necessarily need to be at the end of the file? It doesn't seem so to me, but not sure.